pulley/src/lib.rs

```diff
@@ -815,6 +815,8 @@ macro_rules! for_each_extended_op {
             fmul32 = Fmul32 { operands: BinaryOperands<FReg> };
             /// `low32(dst) = low32(src1) / low32(src2)`
             fdiv32 = Fdiv32 { operands: BinaryOperands<FReg> };
+            /// `low128(dst) = low128(src1) / low128(src2)`
+            vdiv32x4 = Vdiv32x4 { operands: BinaryOperands<VReg> };
```

Comment:
Mind including the `f` here in `f32x4` to clarify that this is a floating-point division instruction as opposed to an integer one?